internal/terraform/exec/exec.go

```diff
@@ -210,65 +112,30 @@ func (e *Executor) FormatterForVersion(v string) (Formatter, error) {
 	}
 
 	if ver.GreaterThanOrEqual(fmtCapableVersion) {
```

Comment:
You could move this version compatibility check to `terraform-exec` if you wanted.

Reply:
this is something @radeksimko upstreaming now. Depending on the outcome of the upstreaming work, this PR may be superseded.

Reply:
https://github.com/hashicorp/terraform-exec/pull/88